src/D2L.Bmx/Okta/OktaApi.cs

```diff
@@ -113,12 +116,10 @@ internal class OktaApi : IOktaApi {
 	}
 
 	async Task<OktaAccountState> IOktaApi.GetAccountsOktaAsync( OktaAuthenticatedState state, string accountType ) {
-		// TODO: Use existing session if it exists in ~/.bmx and isn't expired
-		var sessionResp = await ( this as IOktaApi ).CreateSessionOktaAsync( new SessionOptions( state.OktaSessionToken ) );
 		// TODO: Consider making OktaAPI stateless as well (?)
```

Comment:
ha! this "todo" echos my comments above.
but also this comment should go with the `AddSession` call. now that the call is moved, this comment no longer makes sense here.